feature(scale-tests): reorganize scale tests location

soyacz opened the pull request:
Reorganization of scale tests - moving to separate directory.

## PR pre-checks (self review)
<!--- PR should be created as Draft, when CI finished and relevant checkboxes selected, add reviewers and then click on "Ready for review" button.-->
<!--- Put an `x` in all the boxes that apply or create PR and then click on all relevant checkboxes: -->
- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

fruch commented on jenkins-pipelines/features-scale-450-500-cluster-test.jenkinsfile:
```diff
@@ -8,7 +8,7 @@ longevityPipeline(
     test_name: 'grow_cluster_test.GrowClusterTest.test_grow_x_to_y',
-    test_config: '''["test-cases/features/scale-cluster.yaml", "configurations/scale-450-500.yaml"]''',
+    test_config: '''["test-cases/scale/scale-cluster.yaml", "configurations/scale-450-500.yaml"]''',
 
```
Maybe also in that he configuration folder you want a scale folder ?

soyacz replied:
yes, configurations are quite organized. done. thanks.

fruch reviewed (approved):
LGTM

fgelcer reviewed (dismissed):
LGTM..

@roydahan @fruch , don't we want to move the pipeline files into its own folder as well? so we will have the pipelines also organised into subjects, not only the configuration files..

fruch commented:
> LGTM..
> 
> @roydahan @fruch , don't we want to move the pipeline files into its own folder as well? so we will have the pipelines also organised into subjects, not only the configuration files..

From my POV, I don't really mind one way or the other, but if those are changed
need to make sure
https://github.com/scylladb/scylla-cluster-tests/blob/master/sct.py#L1192

is updated to create the right folder structure.

roydahan commented:
We don't need to move the pipelines to a different folder, BUT I think we need to rename them and as @fruch mentioned to change the the script that creates the jobs accordingly.

@soyacz let's try to do it ASAP, because we want to start working on 5.0 with Argus and it's better we will have them in Argus with the correct location and category, so we won't have duplications of "old" and "new" jobs.

@bentsi / @k0machi FYI

soyacz commented:
I've renamed the files and added another group to `create_test_release_jobs` function.
@fruch please check if it looks fine.

We need to remember to fix jenkinsfiles names in master jenkins folder jobs after merging this commit.

fruch reviewed (approved):
LGTM

roydahan reviewed (approved):
Looks good.
I'll merge and run the job to recreate 5.0 jobs.

roydahan commented:
Why checks are failing?
